dart/test/server_test.dart

```diff
@@ -279,7 +279,7 @@ void main() {
         requestsController.add(
             rpcRequest('streamListen', params: {'streamId': serviceStream}));
         requestsController.add(
-            rpcRequest('_registerService', params: {'service': serviceId}));
+            rpcRequest('registerService', params: {'service': serviceId}));
```

Comment:
I assume the dev sdk now has this change?

Reply:
That's a good question. The dev branch doesn't seem to have it quite yet though...